style.css

```diff
@@ -0,0 +1,127 @@
+.flex-container {
+  display: flex;
+  width: 100px;
+  flex-wrap: wrap;
+}
+
+h1 {
+  position: static;
+  font-family: serif;
```

Comment:
- [ ] Hi, you've done really well to implement most of the required changes. as the previous reviewer suggest please make sure to use the correct fonts as the [Figma Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471). please 🙏 pay close attention the font's in the Figma. The font use is a combination of `Crete Round` and `Roboto`. please implement this 👍. you are doing really well so far.


Your Design                                                        |  Figma Design                                  |
:-------------------------------------------------------------:|:-----------------------------------------------:
![](https://user-images.githubusercontent.com/39523312/143448174-5ae42524-0c61-4037-b199-4971384f3fad.png)  |  ![](https://user-images.githubusercontent.com/39523312/143448182-3e34428d-b8a8-4276-bf35-3fd71465fa60.png)|


_Originally posted by @mmsesay in https://github.com/VTY1999/Portfolio-setup-and-mobile-version-skeleton/pull/1#discussion_r756881548_